internal/services/datafactory/data_factory_data_flow_resource.go

```diff
@@ -243,8 +238,10 @@ func expandDataFactoryDataFlowTransformation(input []interface{}) *[]datafactory
 	for _, v := range input {
```

Comment:
expandDataFactoryDataFlowTransformation and flattenDataFactoryDataFlowTransformation move to internal/services/datafactory/data_factory_data_flow.go